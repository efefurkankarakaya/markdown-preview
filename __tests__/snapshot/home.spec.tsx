import { render } from "@testing-library/react";
import Home from "@/pages/index";

test("If home page is rendered unchanged", async () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
