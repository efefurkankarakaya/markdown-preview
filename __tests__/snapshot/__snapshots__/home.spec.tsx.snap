// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`If home page is rendered unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="innerContainer"
    >
      <div
        class="formatIcons"
      >
        <button
          class="button"
        >
          <img
            alt="bold"
            data-nimg="1"
            decoding="async"
            fetchpriority="high"
            height="40"
            src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
            srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
            style="color: transparent;"
            width="40"
          />
        </button>
      </div>
      <div
        class="editorContainer"
      >
        <textarea
          class="textArea textArea padding margin border"
        >
          # Markdown: A Simple Markup Language

Markdown is a lightweight markup language that allows you to format plain text documents using a simple and easy-to-read syntax. It was created by John Gruber and Aaron Swartz in 2004 with the goal of making it simple to write structured documents that can be easily converted to HTML.

## Headers

You can create headers by using hashtags (\`#\`). The number of hashtags indicates the header level:


## Emphasis

You can emphasize text by using asterisks (\`*\`) or underscores (\`_\`):

- *Italic text* can be created using \`*\` or \`_\`.
- **Bold text** can be created using \`**\` or \`__\`.

## Lists

Markdown supports both ordered and unordered lists:

### Unordered Lists

* Item 1
* Item 2
* Item 3

### Ordered Lists

1. a
2. b

## Links

You can create links with descriptive text:

[Markdown Preview](https://github.com/efefurkankarakaya/markdown-preview) is a free and open source web tool.

## Images

You can include images similarly to links, with an exclamation mark (!) in front:

![Alt text](https://upload.wikimedia.org/wikipedia/commons/thumb/d/d9/Markdown.png/320px-Markdown.png)

## Blockquotes

To create blockquotes, use the &gt; symbol:

&gt; This is a blockquote.

## Code

To format inline code, wrap it with backticks (\`\`):

Use \`print("Hello, world!")\` to display text.

For code blocks, use triple backticks:
\`\`\`
def example_function():
    print("This is a code block")
\`\`\`

## Horizontal Rules

You can create horizontal rules with three or more hyphens, asterisks, or underscores:

---

        </textarea>
        <div
          class="outputArea padding margin border"
          id="outputField"
        >
          <h1>
            Markdown: A Simple Markup Language
          </h1>
          

          <p>
            Markdown is a lightweight markup language that allows you to format plain text documents using a simple and easy-to-read syntax. It was created by John Gruber and Aaron Swartz in 2004 with the goal of making it simple to write structured documents that can be easily converted to HTML.
          </p>
          

          <h2>
            Headers
          </h2>
          

          <p>
            You can create headers by using hashtags (
            <code>
              #
            </code>
            ). The number of hashtags indicates the header level:
          </p>
          

          <h2>
            Emphasis
          </h2>
          

          <p>
            You can emphasize text by using asterisks (
            <code>
              *
            </code>
            ) or underscores (
            <code>
              _
            </code>
            ):
          </p>
          

          <ul>
            

            <li>
              <em>
                Italic text
              </em>
               can be created using 
              <code>
                *
              </code>
               or 
              <code>
                _
              </code>
              .
            </li>
            

            <li>
              <strong>
                Bold text
              </strong>
               can be created using 
              <code>
                **
              </code>
               or 
              <code>
                __
              </code>
              .
            </li>
            

          </ul>
          

          <h2>
            Lists
          </h2>
          

          <p>
            Markdown supports both ordered and unordered lists:
          </p>
          

          <h3>
            Unordered Lists
          </h3>
          

          <ul>
            

            <li>
              Item 1
            </li>
            

            <li>
              Item 2
            </li>
            

            <li>
              Item 3
            </li>
            

          </ul>
          

          <h3>
            Ordered Lists
          </h3>
          

          <ol>
            

            <li>
              a
            </li>
            

            <li>
              b
            </li>
            

          </ol>
          

          <h2>
            Links
          </h2>
          

          <p>
            You can create links with descriptive text:
          </p>
          

          <p>
            <a
              href="https://github.com/efefurkankarakaya/markdown-preview"
            >
              Markdown Preview
            </a>
             is a free and open source web tool.
          </p>
          

          <h2>
            Images
          </h2>
          

          <p>
            You can include images similarly to links, with an exclamation mark (!) in front:
          </p>
          

          <p>
            <img
              alt="Alt text"
              src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d9/Markdown.png/320px-Markdown.png"
            />
          </p>
          

          <h2>
            Blockquotes
          </h2>
          

          <p>
            To create blockquotes, use the &gt; symbol:
          </p>
          

          <blockquote>
            

            <p>
              This is a blockquote.
            </p>
            

          </blockquote>
          

          <h2>
            Code
          </h2>
          

          <p>
            To format inline code, wrap it with backticks (\`\`):
          </p>
          

          <p>
            Use 
            <code>
              print("Hello, world!")
            </code>
             to display text.
          </p>
          

          <p>
            For code blocks, use triple backticks:
          </p>
          

          <pre>
            <code>
              def example_function():
    print("This is a code block")

            </code>
          </pre>
          

          <h2>
            Horizontal Rules
          </h2>
          

          <p>
            You can create horizontal rules with three or more hyphens, asterisks, or underscores:
          </p>
          

          <hr />
          

        </div>
      </div>
    </div>
  </div>
</div>
`;
